templates/sync-function/validation-module.js

```diff
@@ -314,7 +318,7 @@ function validationModule(utils, simpleTypeFilter, typeIdValidator) {
               break;
             default:
               // This is not a document validation error; the item validator is configured incorrectly and must be fixed
-              throw { forbidden: 'No data type defined for validator of item "' + buildItemPath(itemStack) + '"' };
+              throw new Error('No data type defined for validator of item "' + buildItemPath(itemStack) + '"');
```

Comment:
This probably should have been a 500 Internal Server Error from the start. Well, now it is.